src/Administration/Resources/app/administration/src/module/sw-category/snippet/de-DE.json

```diff
@@ -67,8 +67,7 @@
         "visible": "In der Navigation ausblenden",
         "imageLabel": "Anzeigebild",
         "descriptionLabel": "Beschreibung",
-                "descriptionPlaceholder": "Gib eine Beschreibung ein..."
-
+        "descriptionPlaceholder": "Gib eine Beschreibung ein..."
```

Comment:
Please insert an empty space between "ein" and the ellipsis. Without it, it means that the word "ein" isn't complete, like being interrupted, while speaking.

Reply:
Arrr, this point was on my ToDo, too. Will check all occurrences

Reply:
Nice one. Thanks for your work :+1:

Reply:
Just changed it in the related files, other modules will be fixed, when I check them these days, too. :-)